File: core/src/main/kotlin/com/tsobu/ona/core/dto/forms/dataval/WeedAssessmentP1.java

package com.tsobu.ona.core.dto.forms.dataval;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class WeedAssessmentP1 {

    @JsonProperty("weedAssessment_P1/index_P1")
    public String weedAssessmentP1IndexP1;
    @JsonProperty("weedAssessment_P1/weedpic_P1")
    public String weedAssessmentP1WeedpicP1;
    @JsonProperty("weedAssessment_P1/weedCover_P1")
    public int weedAssessmentP1WeedCoverP1;
    @JsonProperty("weedAssessment_P1/weedsHeight_P1")
    public int weedAssessmentP1WeedsHeightP1;
    @JsonProperty("weedAssessment_P1/cassavaHeight_P1")
    public int weedAssessmentP1CassavaHeightP1;

}

File: forms/src/main/kotlin/com/tsobu/ona/forms/dataval/WeedAssessmentP1.kt
package com.tsobu.ona.forms.dataval

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonProperty

@JsonInclude(JsonInclude.Include.NON_NULL)
class WeedAssessmentP1 {
    @JsonProperty("weedAssessment_P1/index_P1")
    var indexP1: String? = null

    @JsonProperty("weedAssessment_P1/weedpic_P1")
    var weedPicP1: String? = null

    @JsonProperty("weedAssessment_P1/weedCover_P1")
    var weedCoverP1 :String?=null

    @JsonProperty("weedAssessment_P1/weedsHeight_P1")
    var weedsHeightP1 :String?=null

    @JsonProperty("weedAssessment_P1/cassavaHeight_P1")
    var cassavaHeightP1 :String?=null
}